examples/login_node_example/Readme.txt

```diff
@@ -0,0 +1,46 @@
+To Create a user using Freeipa
```

Comment:
Can we move this to `/docs` and turn it into a markdown file?

Reply:
@lwilson, This would be a better approach. I'll get it added into docs. I'll close this PR as well.